packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -533,7 +532,7 @@ class IOSSimulator extends Device {
 
   @override
   Future<bool> stopApp(
-    ApplicationPackage app, {
+    ApplicationPackage? app, {
```

Comment:
This is where the nullability mismatch caused the runtime test failure seen during the roll
https://fusion2.corp.google.com/invocations/68e930bd-fdd0-486f-9b8d-dec7798677c9/targets/%2F%2Fmobile%2Fflutter%2Ftests%2Fapp:app_basic_runner_attach_ios_test_IPHONE_11_13_0/log
Since `covariant` has been removed from the base class signature, there was correctly an error when this wasn't nullable.